.buildkite/scripts/steps/ess_start.sh

```diff
@@ -1,21 +1,25 @@
 #!/usr/bin/env bash
 set -euo pipefail
-
-source .buildkite/scripts/common2.sh
+source .buildkite/scripts/steps/stable_ess_version.sh
+source .buildkite/scripts/common-integration.sh
 source .buildkite/scripts/steps/ess.sh
 source .buildkite/scripts/steps/fleet.sh
 
-OVERRIDE_STACK_VERSION="$(cat .package-version)"
-OVERRIDE_STACK_VERSION=${OVERRIDE_STACK_VERSION}"-SNAPSHOT"
-
-ess_up $OVERRIDE_STACK_VERSION
+echo "~~~ Getting stable stack version"
+DEFAULT_STACK_VERSION="$(cat .package-version)-SNAPSHOT"
+STABLE_ESS_VERSION="$(getStableEssSnapshotForBranch)-SNAPSHOT"
+ess_up $DEFAULT_STACK_VERSION $STABLE_ESS_VERSION
+echo "ES_HOST: ${ELASTICSEARCH_HOST}"
```

Comment:
Is this the place to print `ELASTICSEARCH_HOST`?

Reply:
It used several times for debugging. I think I can delete it